cabal.project

```diff
@@ -0,0 +1,5 @@
+packages:
```

Comment:
This is needed because haskell-ci wants you to pass a single cli argument describing your project (either cabal file or cabal.project describing multi-package project).